pkg/cloudprovider/providers/openstack/openstack_volumes.go

```diff
@@ -268,6 +294,27 @@ func (os *OpenStack) DetachDisk(instanceID, volumeID string) error {
 	return nil
 }
 
+// ExpandVolume expands the size of specific cinder volume (in GiB)
+func (os *OpenStack) ExpandVolume(volumeID string, newSize int) error {
```

Comment:
Calling this function should be idempotent. I am wondering if there is a way we can describe the disk and check the size before deciding if resizing is required.

Reply:
Done